README.md

```diff
@@ -97,3 +98,42 @@ since the commit hash is used as the image tag.
    ```bash
       hubploy deploy <hub-name> hub prod
    ```
+
+## How to use an unreleased helm chart
+
+Sometimes you might need to use the latest version of a helm chart and that isn't yet released.
+The following steps shows how to do so for the DaskHub project, but the pattern
+should be replicable for others.
+
+1. Clone the repo holding the latest version of the helm-chart and checkout the wanted branch:
+
+   *DaskHub example:*
+
+   ```bash
+      git clone https://github.com/dask/helm-chart.git
+      git checkout master
+   ```
+2. Inside the `yaml` file holding the dependencies, add the local path to the helm-chart of the project
+you just checked out and use the version inside the `chart.yaml` or run `chartpress` to generate one if
+that is not available:
```

Comment:
```suggestion
2. Inside the `yaml` file holding the dependencies, either `requirements.yaml` or `Chart.yaml`, reference the version of the unpackaged Helm chart in the locally cloned git repository. This version should be available in `Chart.yaml`, but if it isn't, it may be because it is using `chartpress` to set it, then you need to run `chartpress` in that repository first.
```

I wanted to iterate on this sentence a bit, perhaps you can help adjust my attempt to something that makes things clear?

Reply:
I edited your comment, hope that's ok. What do you think about it? :D

Reply:
Nice! I did another iteration again on the last sentence, feel free to edit directly or do as you please!

```diff
-If no version is available it may be because it is using `chartpress` to set it, so you need to run `chartpress` in that repository first to generate it.
+If the version is set to be a placeholder or outdated, it may be because it is maintained using `chartpress`, then run `chartpress` in that repository first to update it.
```

```suggestion
2. Reference the version of the unpackaged Helm chart (of the locally cloned git repository) in the dependencies `yaml` file of your project (either `requirements.yaml` or `Chart.yaml`). This version should be available in the`Chart.yaml` file (of the cloned repository). If the version is set to be a placeholder or outdated, it may be because it is maintained using `chartpress`, then run `chartpress` in that repository first to update it.
```

Reply:
No, that's perfect :heart: Thanks Erik, I really like how it turned out.

Reply:
Thanks for iterating on it with me and working on this documentation @GeorgianaElena! :sunflower: